src/app/audiologist-login/audiologist-login.component.ts

```diff
@@ -14,11 +15,17 @@ import { ThsDataService } from '../services/ths-data.service';
 
 export class AudiologistLoginComponent {
   // Added audiologistUserName & changed (patientId --> audiologistID)
-  public audiologistUserName: string = ''; 
+  public audiologistUserName: string = '';
   public audiologistID: string = '';
```

Comment:
This should be renamed to audiologistPassword to avoid confusion, now that authentication takes username and password.(Admittedly, I should've done this when I changed it)